events.lisp

```diff
@@ -152,29 +150,26 @@
   (dformat 2 "UNMAP: ~s ~s ~a~%" send-event-p (not (xlib:window-equal event-window window)) (find-window window))
   (unless (and (not send-event-p)
                (not (xlib:window-equal event-window window)))
-    (let ((window (find-window window)))
+    (when-let ((window (find-window window)))
       ;; if we can't find the window then there's nothing we need to
```

Comment:
The comment would now belong on top of the when-let, but I think we can remove it altogether. It is redundant with the code.